doc/source/user_guide/ug_case_settings.rst

```diff
@@ -275,6 +275,9 @@ Table of namelist options
    "","", "``-1``", "Transport Disabled", ""
    "","``basalstress``", "true/false", "use basal stress parameterization for landfast ice", ""
    "","``k1``", "real", "1st free parameter for landfast parameterization", "8."
+   "","``k2``", "real", "2nd free parameter (N/m^3) for landfast parameterization", "15."
+   "","``alphab``", "real", ":math:`\alpha_{b}` factor in :cice:`Lemieux16`", "20."
```

Comment:
The Lemieux16 is not being resolved properly in the documentation.  See https://cice-consortium-cice--404.org.readthedocs.build/en/404/user_guide/ug_case_settings.html#table-of-namelist-options.  Could this be corrected?